pkg/security/probe/probe.go

```diff
@@ -1188,8 +1187,28 @@ func (p *Probe) GetDebugStats() map[string]interface{} {
 	return debug
 }
 
-// NewRuleSet returns a new rule set
-func (p *Probe) NewRuleSet(eventTypeEnabled map[eval.EventType]bool) *rules.RuleSet {
+// NewPolicySet returns a new policy set
+func (p *Probe) NewPolicySet(eventTypeEnabled map[eval.EventType]bool) *rules.PolicySet {
+	ruleOpts, evalOpts := rules.NewEvalOpts(eventTypeEnabled)
+
+	ruleOpts.WithLogger(seclog.DefaultLogger)
+	ruleOpts.WithSupportedDiscarders(SupportedDiscarders)
+	ruleOpts.WithReservedRuleIDs(events.AllCustomRuleIDs())
+
+	eventCtor := func() eval.Event {
+		return &model.Event{
+			FieldHandlers: p.fieldHandlers,
+		}
+	}
+
+	defaultRuleSet := rules.NewRuleSet(NewModel(p), eventCtor, ruleOpts, evalOpts)
+	threatScoreRuleSet := rules.NewRuleSet(NewModel(p), eventCtor, ruleOpts, evalOpts)
+
+	return rules.NewPolicySet(defaultRuleSet, threatScoreRuleSet)
```

Comment:
I would keep this logic of keeping to ruleset in the module package. We are trying to move all the CWS logic in the module package.